addon/templates/components/uni-modal.hbs

```diff
@@ -1,15 +1,17 @@
 {{#if isOpen}}
-<div class="uni-modal">
+<div class="uni-modal {{if isSignup 'uni-modal--signup'}}">
```

Comment:
We can discuss with @montoias , but in my opinion, we should have a uni-signup-modal and uni-login-modal (or just an uni-auth-modal) instead of attaching everything to the more generic "uni-modal"